.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
package-lock.json should be added to the repo and not be in .gitignore. See that the file has been added by it is also present in .gitigonre